src/test/unit_tests/jest_tests/test_nc_nsfs_schema_validation.test.js

```diff
@@ -708,10 +736,24 @@ describe('schema validation NC NSFS account', () => {
 
     });
 
+});
+
+describe('schema validation NC NSFS account with wrong types', () => {
+    const account_name = 'account1';
+    const account_id = 'account1';
+    const account_email = 'account1@noobaa.io';
+    const access_key = 'GIGiFAnjaaE7OKD5N7hA';
+    const secret_key = 'U2AYaMpU3zRDcRFWmvzgQr9MoHIAsD+3oEXAMPLE';
+    const creation_date = new Date('December 17, 2023 09:00:00').toISOString();
+    const nsfs_account_config_uid_gid = {
+        uid: 1001,
+        gid: 1001,
+    };
```

Comment:
Those variables were declared in a higher level of `describe`, no need to declare them again and override the previous declaration. As said, I changed the file.